src/lucky_web/tags/text_helpers.cr

```diff
@@ -0,0 +1,217 @@
+module LuckyWeb::TextHelpers
+  @@_cycles = Hash(String, Cycle).new
+
+  def truncate(text, length = 30, omission = "...", separator = nil, escape = false, blk : Nil | Proc = nil)
+    if text
+      content = truncate_text(text, length, omission, separator)
+      content = escape ? HTML.escape(content) : content
+      content += blk.call.to_s if blk.is_a?(Proc) && text.size > length
+      content
+    end
+  end
+
+  def truncate(text, length = 30, omission = "...", separator = nil, escape = true, &block : -> _)
```

Comment:
Can you add type specific types for all these `text : String, length : Int32 = 30`, etc.? It makes the docs read a bit better because people can more easily see the types